Gopkg.toml

```diff
@@ -95,15 +95,16 @@ required = [
   name = "golang.org/x/oauth2"
   revision = "cdc340f7c179dbbfa4afd43b7614e8fcadde4269"
 
-# TODO why is this overridden?
-[[override]]
+# Constrain the version of knative/pkg we would like to import.
+# This controls when we upgrade apis independently of Serving.
+[[constraint]]
```

Comment:
:+1: Now when we ask for a different version of knative/pkg than serving asks for, dep will tell us. Prevents a silent change until we actually intend it.